fla/ops/simple_gla/fused_chunk.py

```
@@ -0,0 +1,601 @@
+            q=q,
+            k=k,
+            v=v,
+            g=g,
+            g_gamma=g_gamma,
+            do=do,
+            scale=ctx.scale,
+            initial_state=initial_state,
+            dht=dht,
+            cu_seqlens=ctx.cu_seqlens,
+            chunk_size=ctx.chunk_size
+        )
```

Pylint (3.3.7)

[error] Unexpected keyword argument 'g' in function call

(E1123)